codespell_lib/data/dictionary.txt

```diff
@@ -30355,7 +30376,7 @@ travesed->traversed
 traveses->traverses
 travesing->traversing
 tre->tree
-treadet->treated
+treadet->treated, treaded,
```

Comment:
Possibly threaded too, but might be a stretch.

Reply:
972a761